tidb-configuration-file.md

```diff
@@ -609,14 +609,14 @@ Configuration items related to performance.
 
 ### `concurrently-init-stats` <span class="version-mark">New in v8.1.0 and v7.5.2</span>
 
-+ Controls whether to initialize statistics concurrently during TiDB startup.
-+ Default value: `false`
++ Controls whether to initialize statistics concurrently during TiDB startup. Only takes effect when [`lite-init-stats`](/tidb-configuration-file.md#lite-init-stats-new-in-v710) is set to `false`.
++ Default value: `true`
```

Comment:
```suggestion
+ Default value: `false` for versions earlier than v8.2.0, `true` for v8.2.0 and later versions.
```

Reply:
No, this still holds true even in v8.2.0. See: https://github.com/Rustin170506/tidb/blob/821e491a20fbab36604b36b647b5bae26a2c1418/pkg/config/config.go#L1022